e2e_tests/tests/experiment/experiment.py

```diff
@@ -413,13 +413,13 @@ def wait_for_experiment_workload_progress(
 def experiment_has_completed_workload(sess: api.Session, experiment_id: int) -> bool:
     trials = experiment_trials(sess, experiment_id)
 
-    if not any(trials):
-        return False
+    if any(trials):
```

Comment:
(Not a blocker) I know this isn't your code in the first place. I'm confused about why we need to wrap the for loop with `if any(trails)`. `any(trails)` returns False:
1. `TrialPlusWorkload` defines `__boo__()` method returns False or `__len__()` method returns 0. We don't define these methods. 
2. `trails` is empty. The for loop doesn't execute if `trials` is empty.

Reply:
I was just simplifying the code a little bit - I think functionally it should be the same as what it was before because we return `False` if any(trials) is false anyways.

Reply:
I think we are on the same page in terms of functionality. I'm just not convinced that we need it in the first place.

Reply:
ohhh i see, ok I just checked and pushed an update. If it passes the tests I'll merge!